projects/swimlane/ngx-ui/src/lib/styles/colors/shadow-colors.scss

```diff
@@ -0,0 +1,123 @@
+$shadow-key-umbra-opacity: 0.2;
```

Comment:
Not sure about the naming of this file. It has more than just colors right?

Reply:
Yeah you're right. Will rename to "shadow-variables"